src/vstart.sh

```diff
@@ -17,8 +17,8 @@ quoted_print() {
 }
 
 debug() {
-  if [ -w /dev/tty -a ! -t 2 ]; then
-    "$@" | tee /dev/tty >&2
+  if [ -w /dev/stderr -a ! -t 2 ]; then
```

Comment:
I tried again and now it really works (although I'm not sure if the resulting behaviour is the same as what @batrick expected when he made this change):
```suggestion
  if [ -w /dev/tty -a -t 1 ]; then
```

Reply:
You're right, the ``! -t 2`` does not make sense when using ``/dev/stderr``. The problem is that Jenkins does not have a ``/dev/tty`` device. I think @batrick needs to adapt this code to let it work as he expects to do so.